python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -768,9 +768,10 @@ def extract_subgraph(self,
             The name of the property whose values will be used as weights on
             the returned Graph. If not specified, the returned Graph will be
             unweighted.
-        allow_multi_edges : bool
-            If True, multiple edges should be used to create the return Graph,
-            otherwise multiple edges will be detected and an exception raised.
+        check_multi_edges : bool (default is True)
+            When True and create_using argument is given and not a MultiGraph,
+            this will perform an expensive check to verify that the edges in
+            the edge dataframe do not form a multigraph with duplicate edges.
```

Comment:
Minor: since I think we're primarily using the term "expensive check" for the `do_expensive_check` debug APIs now, maybe we should avoid using it in help strings for other options like this one.  Unless, we mean to explicitly tell the user this particular check is especially expensive? If so, we could instead add a sentence warning the user this option could be slow.
```suggestion
            this will perform a check to verify that the edges in the edge
            dataframe do not form a multigraph with duplicate edges.
```